app/javascript/src/components/GameLogEntry.vue

```diff
@@ -1,17 +1,132 @@
 <template>
-  <div class="border border-black rounded p-2 m-2">
+  <div v-if="Array.isArray(event)" class="border border-black rounded p-2 m-2 rondel">
```

Comment:
Ah, my recommendation above would break this logic. I think it's worth investigating this a small bit further. Can we turn our conditional on something other than the array-ness of `event`? Perhaps `event.type` would be a more direct way to get the logic to work?

Reply:
As mentioned before it's worth changing this. :+1: